src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -122,6 +125,8 @@ function cloneUpdateQueue(alt : Fiber, fiber : Fiber) : UpdateQueue | null {
   altQueue.first = sourceQueue.first;
   altQueue.last = sourceQueue.last;
   altQueue.hasForceUpdate = sourceQueue.hasForceUpdate;
+  altQueue.callbackList = sourceQueue.callbackList;
+  altQueue.isProcessing = sourceQueue.isProcessing;
```

Comment:
No bug, just noticed it wasn't being copied and I added it for consistency's sake. Although actually, `callbackList`, `isProcessing`, and `hasForceUpdate` can be reset when cloning. I'll do that instead. https://github.com/facebook/react/pull/8728#discussion_r95282877